pom.xml

```diff
@@ -2023,6 +2023,46 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>com.github.ekryd.sortpom</groupId>
+                    <artifactId>sortpom-maven-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <executions>
+                        <execution>
+                            <id>sortpom-warn</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Warn</verifyFail>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>sortpom-fail</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Stop</verifyFail>
+                            </configuration>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <createBackupFile>false</createBackupFile>
+                        <keepBlankLines>true</keepBlankLines>
+                        <nrOfIndentSpace>4</nrOfIndentSpace>
+                        <sortOrderFile>${air.main.basedir}/.mvn/sortpom/sortorder.xml</sortOrderFile>
+                        <sortDependencyManagement>scope,groupId,artifactId</sortDependencyManagement>
+                        <sortDependencies>scope,groupId,artifactId</sortDependencies>
+                        <sortDependencyExclusions>groupId,artifactId</sortDependencyExclusions>
+                        <expandEmptyElements>false</expandEmptyElements>
+                        <verifyFailOn>Strict</verifyFailOn>
+                        <violationFilename>${air.main.basedir}/target/sortpom-violation.xml</violationFilename>
```

Comment:
Do we need this? Also, this seems wrong because

1. The plugin runs for each module, but this will put the violation file in the root module target directory.
2. You shouldn't hard-code `target` as that can theoretically change. The correct way is the `project.build.directory` property.

Reply:
Will remove violation file for now.